sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -795,20 +840,65 @@ def waiting_time_compensation_with_sleep(time_start, time_to_wait):
     if time_diff < time_to_wait:
         time.sleep(time_to_wait - time_diff)
 
-# Update port SFP status table on receiving SFP change event
+# Update port SFP status table for SW fields on receiving SFP change event
 
 
-def update_port_transceiver_status_table(logical_port_name, status_tbl, status, error_descriptions='N/A'):
+def update_port_transceiver_status_table_sw(logical_port_name, status_tbl, status, error_descriptions='N/A'):
     fvs = swsscommon.FieldValuePairs([('status', status), ('error', error_descriptions)])
     status_tbl.set(logical_port_name, fvs)
 
+# Update port SFP status table for HW fields
```

Comment:
You have created 2 different functions to update *status_table_hw and *status_table_sw but does it update same status table? what are variables here?

Reply:
one is for updating SW fields which are eeprom-fetched ('status' 'error' ); one is for updating HW fields which are eeprom-fetched. I updated in the PR description about which are SW fields, which are HW fields.